docs/_docs/maintaining/merging-a-pull-request.md

```diff
@@ -30,12 +30,12 @@ The merge request is made up of three things:
 
 1. `@jekyllbot:` – this is the prefix our bot looks for when processing commands
 2. `merge` – the command
-3. `+dev` – the category to which the changes belong
+3. `+dev` – the category to which the changes belong. The `+` is optional.
 
-The categories match the H3's in the history/changelog file, and they are:
+The categories match the headings in the `History.markdown` file, and they are:
 
 1. Major Enhancements (`+major`) – major updates or breaking changes to the code which necessitate a major version bump (v3 ~> v4)
-2. Minor Enhancements (`+minor`) – minor updates (feature, enhancement) which necessitate a minor version bump (v3.1 ~> v3.2)
+2. Minor Enhancements (`+minor`) – minor updates (features, enhancements) which necessitate a minor version bump (v3.1 ~> v3.2)
```

Comment:
I think `feature` and `enhancement` refers specifically to the labels, and need not be pluralized.